.travis.yml

```diff
@@ -40,6 +46,8 @@ script:
   - make bdddeps
   # Running all go tests
   - make test
+  # Running Trivy Security check
+  - make trivy-security-check
```

Comment:
This uses the image. By this point in the build, the image has not been created

Reply:
Refactored